include/System/ParamInst.hpp

```diff
@@ -1,13 +1,35 @@
 #ifndef PARAM_INST_HPP
 #define PARAM_INST_HPP
 
+#include "JSystem/JDrama/JDRNameRef.hpp"
 #include "System/BaseParam.hpp"
 
+#define PARAM_INIT(member, value)                                              \
+	member(this, JDrama::TNameRef::calcKeyCode("#member"), "#member", value)
+
 template <typename T> class TParamT : public TBaseParam {
 public:
+	TParamT(TParams* params, unsigned short code, const char* paramName,
+	        T defaultValue)
+	    : TBaseParam(params, code, paramName)
+	    , value(defaultValue)
+	{
+	}
+
 	void load(JSUMemoryInputStream& stream);
+	T get() const { return value; }
```

Comment:
please try to mark fabricated inlines with //fabricated for future reference

Reply:
Sounds smart, makes it easier to distinguish between known symbols and guessed ones. i assume?